.travis.yml

```diff
@@ -9,6 +9,12 @@ cache:
   directories:
     - $HOME/sdk
 
+before_install:
+  - sudo apt-get install -y xsltproc
```

Comment:
I'd rather add a host package. We can't assume everyone has this.

Reply:
The mentioned PR https://github.com/openwrt/packages/pull/10261 seems to be stalled. I don't mind to drop this commit as soon as the host package is merged.

Reply:
Dropped.